package.json

```diff
@@ -46,39 +40,45 @@
     "@metalsmith/in-place": "^5.0.0",
     "@metalsmith/layouts": "^2.7.0",
     "@metalsmith/permalinks": "^2.5.1",
+    "@metalsmith/postcss": "^5.4.1",
     "@metalsmith/sass": "^1.6.0",
     "@rollup/plugin-commonjs": "^25.0.3",
     "@rollup/plugin-node-resolve": "^15.1.0",
     "@rollup/plugin-terser": "^0.4.3",
-    "accessible-autocomplete": "^2.0.4",
+    "autoprefixer": "^10.4.14",
     "browser-sync": "2.29.3",
+    "connect": "^3.7.0",
     "cross-env": "^7.0.3",
     "eslint": "^8.45.0",
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-n": "^16.0.1",
     "eslint-plugin-promise": "^6.1.1",
     "glob": "^10.3.3",
+    "gray-matter": "^4.0.2",
+    "highlight.js": "^11.8.0",
     "html-validate": "^8.1.0",
     "hyperlink": "^5.0.4",
-    "iframe-resizer": "^4.3.6",
     "jest": "^29.6.1",
     "jest-axe": "^8.0.0",
     "jest-environment-jsdom": "^29.6.1",
     "jest-puppeteer": "^9.0.0",
     "js-beautify": "^1.14.9",
     "jstransformer-marked": "^1.4.0",
     "jstransformer-nunjucks": "^1.2.0",
-    "marked": "^5.0.2",
+    "marked": "^5.1.2",
```

Comment:
We've missed the following release (no idea why)

* [Marked v5.0.3](https://github.com/markedjs/marked/releases/tag/v5.0.3)
* [Marked v5.0.4](https://github.com/markedjs/marked/releases/tag/v5.0.4)
* [Marked v5.0.5](https://github.com/markedjs/marked/releases/tag/v5.0.5)
* [Marked v5.1.0](https://github.com/markedjs/marked/releases/tag/v5.1.0)
* [Marked v5.1.1](https://github.com/markedjs/marked/releases/tag/v5.1.1)
* [Marked v5.1.2](https://github.com/markedjs/marked/releases/tag/v5.1.2)

No build output changes

Reply:
@domoscargin Wondered if you knew why it was held back? Can't see any config

The [recent Metalsmith plugin updates](https://github.com/alphagov/govuk-design-system/pull/2966) didn't unblock Dependabot either, as it [still logged](https://github.com/alphagov/govuk-design-system/network/updates/700298719):

```console
updater | 2023/07/25 09:50:51 INFO <job_700298719> Checking if marked 5.0.2 needs updating
updater | 2023/07/25 09:50:51 INFO <job_700298719> Ignored versions:
updater | 2023/07/25 09:50:51 INFO <job_700298719>   >= 5.a, < 6 - from @dependabot ignore command
  proxy | 2023/07/25 09:50:51 [350] GET https://registry.npmjs.org:443/marked
  proxy | 2023/07/25 09:50:51 [350] 200 https://registry.npmjs.org:443/marked
updater | 2023/07/25 09:50:51 INFO <job_700298719> All updates for marked were ignored
```

Reply:
Based on `from @dependabot ignore command` I reckon it's a hangover from this:

https://github.com/alphagov/govuk-design-system/pull/2766#issuecomment-1540098297

I'd have expected that to get 'reset' once we manually bump to 5.x, but apparently not…

Reply:
Ahh shall we re-open that?